tvb/analyzers/fragility_lib/execute/singlecore/singlemvar.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np 
+import sys
+import os
+sys.path.append('../../')
+from ...linearmodels.basemodel import BaseWindowModel
+from ...linearmodels.mvarmodel import MvarModel
```

Comment:
modifying `sys.path` isn't appropriate; you're probably just missing some `__init__.py` files in the relevent folders.  Let me know if I should take a look

Reply:
Yes, you are right. I removed this line. I think the 

"from ...linearmodels.basemodel ..." 

should handle it correctly.